linode/manager
# tech story: [M3-8386] - Update storybook to take care of tar vulnerability

## Description 📝
- Updates storybook and related addons to the latest version resolve [this tar vulnerability](https://github.com/linode/manager/security/dependabot/98)

## Changes  🔄
- upgrade Storybook to 8.3.0

## Target release date 🗓️
n/a

## How to test 🧪
- confirm giget dependency (and therefore tar 6.2.0) is removed from yarn.lock
- Run/build storybook locally to confirm no regression was introduced with this update

## As an Author I have considered 🤔

*Check all that apply*
